nestkernel/event.h

```diff
@@ -827,13 +843,20 @@ class GapJEvent : public SecondaryEvent
   static void
   set_syn_id( const synindex synid )
   {
-    synid_ = synid;
+    supported_syn_ids_.push_back( synid );
+  }
+
+  void
+  add_syn_id( const synindex synid )
+  {
+    supported_syn_ids_.push_back( synid );
```

Comment:
The way it is constructed this should indeed never happen. I will add `assert( ... )`